### ISSUE TITLE
MISP Duplicates alert emails for published events (OTX-MISP)

### ISSUE THREAD

**Issue by ndhambi** [MISP/MISP#2562]
MISP sends duplicate alert emails for every pulse it imports from OTX, I have attached an image of my email inbox with the alert inboxes, If you look closely you will see that there is a duplicate for each email. Is this a bug in the code ?

![misp](https://user-images.githubusercontent.com/30654371/31539431-d5c82216-b008-11e7-9ea5-0cddc92f6712.PNG)

**Comment by adulau** [comment]
Are you sure you don't have two accounts going to the same mailbox?

**Comment by ndhambi** [comment]
Yes i am sure, My MISP has 5 users and they all have different emails, i even double checked on the List users tab, all the users receive duplicate emails
